reviews/2020-Network Service Mesh-annual.md

```diff
@@ -0,0 +1,192 @@
+Where a workload ran determined what networking was available to it and the semantics of what networking features
+it could receive.
+
+Network Service Mesh allows workloads to be 'loosely coupled' to the Network Service (or Network Services) that are
+relevant to their particular work independent of the environment they are running in.  Workloads running in different
+clusters hybrid/multi-cloud scenarios can connect to a common Network Service that provides the networking features
+needed for their particular collaboration.
+
+Network Service Mesh does this without disturbing (and taking care not to break) the existing networking provided
+by the runtime.  K8s Networking via CNI is untouched, and continues to work as expected.
+
+Network Service Mesh makes IP networking itself Cloud Native by loosely coupling with the existing 
+immutable infrastrcture in K8s to deliver whatever non-standard Networking needs a workload has with minimal toil.
+
+NSM is a complement to higher level L7 Service Meshes (Linkerd), providing an additional 
+connectivity,security and observability at lower layers.
+
+### Minimal Toil
+The Network Service Mesh enabled developers access Network Services with minimal toil:
+
+* Developers ask for what Network Service they want by name
+* Developers can add metadata to their request for a Network Service with labels.
+* Developers simply get the Network Service they asked for
+
+Developers never have to know or think about 35 year old legacy Networking concepts like:
+
+* Subnets
+* Interfaces
+* Routes
+* IPAM
+
+### Immutable Infrastructure
+
+Network Service Mesh shifts IP networking from being 'Infrastructure' to being a selection of 'Network Services'.
+This allows infrastructure to remain immutable while meeting a much wider variety of requirements.  
+By scoping the selection of 'Network Services' to the granularity of workloads (rather than clusters), Network Service Mesh
+allows different workloads to consume different Network Services that could be conflicting if forced into cluster level
+infrastructure.
+
+## Highlights of Last Year
+###  Community
+   In the last year Network Service Mesh has had contributions from:
+   *  [31 contributors](https://networkservicemesh.devstats.cncf.io/d/22/prs-authors-table?orgId=1&var-period_name=Last%20year&var-repogroup_name=All) - up from 18 contributors when NSM was accepted as a CNCF Sandbox project
+   *  [18 Organizations](https://networkservicemesh.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=commits)
+    
+      Including:
+      *  Cisco
+      *  Ericsson
+      *  Intel
+      *  Red Hat
+      *  SUSE
+      *  VMWare
+
+###  Releases
+   
+Network Service Mesh had [two releases](https://networkservicemesh.io/docs/releases/b/) in the last year:
+* [Andromeda](https://networkservicemesh.io/docs/releases/a/) - May 14, 2019
```

Comment:
Would be great to include information about the features shipped with every release, to the release notes. This information could benefit the users

Reply:
We'll put together a list of changes and link them here. We'll put together a list of action items that need to be done and verified per release. We'll make sure the change list is on that list. Again, thanks for the recommendation here.

Reply:
We have a [PR](https://github.com/networkservicemesh/site/pull/184) out to the community for release notes for v0.2.0.

Reply:
And this [PR](https://github.com/networkservicemesh/site/pull/185) for release notes for v0.1.0

Reply:
@alena1108 Both the mentioned PRs have now been merged and so the links in the annual review here to the two releases should now reflect feature changes since the last release.  Let me know if those look good, I realize we still have some more questions to respond to here, working through them :)